packages/cdktf-cli/bin/cmds/synth.ts

```diff
@@ -22,6 +22,11 @@ class Command implements yargs.CommandModule {
         desc: "Stack to output when using --json flag",
         type: "string",
       })
+      .option("check-code-maker-output", {
```

Comment:
I think this isn't needed since the value specified in `cdktf.json` will be honored.

Reply:
It seems a bit unnatural to me that such a simple flag is an option that can only be specified in a config file. 

The field `checkCodeMakerOutput` seems to be used only in `synth` so far.
So, IMAO, it seems most explicit to remove the field from the Config interface and implement it as a method.
WDYT??

Reply:
I tried to implement it, WDYT??

Reply:
There currently aren't very many arguments to the `synth` command. I believe there is value in keeping it that way, especially since there is already a configuration file.

Also, `codeMakerOutput` is already only present in the configuration file, so I think it makes sense to keep an option corresponding to its use next to it.

All that said, it probably doesn't really matter. Either way it is functionality that needs supporting.

Reply:
We have a pretty similar pattern on other commands (e.g. https://github.com/hashicorp/terraform-cdk/blob/a9879d4726c18138ac556a8387934bf786825c52/packages/cdktf-cli/bin/cmds/deploy.ts#L23-L34). Don't have a strong opinion, I'd be ok with having that flag.

Reply:
I also don't have a strong opinion too. Please decide which way should I go. @jsteinich 

1. revert 3950895 and then remove the flag
2. revert 3950895 and then leave the flag
3. as is ( leave only the flag

My preference is 3, next is 2.